addons/l10n_latam_invoice_document/views/report_templates.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <template id="external_layout_standard" inherit_id="web.external_layout_standard" >
+        <!-- support for custom header -->
+        <div t-attf-class="header o_company_#{company.id}_layout" position="attributes">
+            <attribute name="t-if">not custom_header</attribute>
+        </div>
```

Comment:
@smetl It seems like you added these lay-outs.  Do you see a problem in adding this to the base modules?  This would allow any report to choose to use its own footer/header while the layout of the contents would still be adapted to the lay-out itself.  Here we need a very specific header with all kinds of extra information and a barcode in the footer.

Reply:
@jco-odoo I don't remember well what I have done with that. You could try but probably not in a stable version :x